boards/weact/mini_stm32h7b0/doc/index.rst

```diff
@@ -0,0 +1,238 @@
+.. zephyr:board:: mini_stm32h7b0
+
+Overview
+********
+
+The MiniSTM32H7B0 Core board is a compact development board equipped with
+an STM32H7B0VBT6 microcontroller. It features a variety of peripherals,
+including a user LED and button, a display, and external SPI and QuadSPI
+NOR flash memory.
+
+Key Features
+
+- STM32 microcontroller in LQFP100 package
+- USB OTG or full-speed device
+- 1 user LED
+- User, boot, and reset push-buttons
+- 32.768 kHz and 25MHz HSE crystal oscillators
+- External NOR Flash memories: 64-Mbit Quad-SPI and 64-Mbit SPI
+- Board connectors:
+   - Camera (8 bit) connector
+   - ST7735 TFT-LCD 160 x 80 pixels (RGB565 3-SPI)
+   - microSD |trade| card
+   - USB Type-C Connector
+   - SWD header for external debugger
+   - 2x 40-pin GPIO connector
+
+.. figure:: img/stm32h7xx.webp
+      :align: center
+      :alt: MiniSTM32H7B0 Core Board
+
+      MiniSTM32H7B0 Core Board (Credit: WeAct Studio)
+
```

Comment:
```suggestion
```

The image auto included in the "wikipedia" side bar is enough.

Bonus point if you rename the image file to be the same as the board name

Reply:
fixed.